scp/embedded-install.sh

```diff
@@ -29,7 +31,7 @@ scpChart="${chartRepo}/scp"
 #keycloakChart="shp-embedded-keycloak-0.1.4.tgz"
 #keycloakBootstrapperChart="shp-keycloak-bootstrapper-0.1.4.tgz"
 #scpChart="scp-0.1.8.tgz"
-while getopts "h:t:s:u:p:e:c:o:k:l:i" arg; do
+while getopts "h:t:s:u:p:e:c:o:k:l:ia" arg; do
```

Comment:
```suggestion
while getopts "h:t:s:u:p:e:c:o:k:l:a" arg; do
```
should it just be `a`

Reply:
i is a different arg for scaling istio